docs/common/input.md

```diff
@@ -440,6 +440,7 @@ physics:
   flow:
     mode: swe
     tiny_h: 1e-7
+    h_anuga_reg_parameter: 1.e-3
```

Comment:
What do you think about calling this `h_anuga_regular`, as you've done in `rdyconfigimpl.h`?

Reply:
Done